### DISCUSSION TITLE
Get max homes using CMI-API

### DISCUSSION THREAD

**Discussion by Adreez** [Zrips/CMI#7899] [Q&A] [answered]
Hi 👋 
is it possible to somehow get max homes of a player using CMI-API? 

Thanks in advance for response 😉

**Comment by MrCrazys** [comment, marked as answer]
Yes, you can use the following:
```java
CMI.getInstance().getHomeManager().getMaxHomes(player);
```
Note that you would need to have a `player` variable to use it.
